src/libraries/System.Runtime.Numerics/src/System/Number.BigInteger.cs

```diff
@@ -640,13 +650,13 @@ bool ProcessChunk(ReadOnlySpan<char> chunkDigits, ref Span<uint> currentBuffer)
                 }
             }
 
-            ParsingStatus DivideAndConquer(ref BigNumberBuffer number, out BigInteger result)
+            ParsingStatus DivideAndConquer(ref NumberBuffer number, out BigInteger result)
             {
                 Span<uint> currentBuffer;
                 int[]? arrayFromPoolForMultiplier = null;
                 try
                 {
-                    totalDigitCount = Math.Min(number.digits.Length - 1, numberScale);
+                    totalDigitCount = Math.Min(number.DigitsCount, numberScale);
```

Comment:
Is `DigitsCount` equal to `number.digits.Length - 1` here?

Reply:
Tests are passing so I assume nothing is wrong. `digits` has a trailing zero in the comment.